### PR TITLE
Call Queue Cmdlet Docs

### PR DESCRIPTION
**Pull request by mattwlam**
Call Queue cmdlet documentation

### CODE REVIEW

**Comment by mattwlam** [comment]
@kenwith

File: skype/skype-ps/skype/Get-CsCallQueue.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+# Get-CsCallQueue
+
+## SYNOPSIS
+The Get-CsCallQueue cmdlet returns the identified Call Queues.
+
+## SYNTAX
+
+```
+Get-CsCallQueue [-BypassDualWrite <Object>] [-DomainController <Object>] [-Force] [-Identity <Object>]
```
Comment (bharatnamburi) [inline skype/skype-ps/skype/Get-CsCallQueue.md]: We are missing many parameters here like NameFilter, First, Skip

File: skype/skype-ps/skype/Get-CsCallQueue.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ [-Tenant <Object>] [-AsJob] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-CsCallQueue cmdlet lets you retrieve information about the Call Queues in your organization. Call Queue output contains statistical data on the number of active calls that are in the queue.
+
+If called without parameters, Get-CsCallQueue returns a collection of all Call Queues configured to be used in your organization.
+
+The Get-CsCallQueue cmdlet may suggest additional steps required to complete the Call Queue setup.
```
Comment (bharatnamburi) [inline skype/skype-ps/skype/Get-CsCallQueue.md]: I dont think this is relevant in new cmdlets. we can skip this

File: skype/skype-ps/skype/Get-CsCallQueue.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+## PARAMETERS
+
+### -DomainController
+PARAMVALUE: Fqdn
+
+```yaml
+Type: Object
+Parameter Sets: (All)
+Aliases: DC
+Applicable: Skype for Business Online
```
Comment (bharatnamburi) [inline skype/skype-ps/skype/Get-CsCallQueue.md]: trying to understand this field, is it required to mention "Skype For Business Online"

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+[-TimeoutAction <Microsoft.Rtc.Management.Hosted.HuntGroup.Models.TimeoutAction>] [-TimeoutActionTarget <System.Guid>] [-TimeoutThreshold <Int16>] 
+[-RoutingMethod <Microsoft.Rtc.Management.Hosted.HuntGroup.Models.RoutingMethod>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The New-CsCallQueue cmdlet creates a new Call Queue.
+
+MusicOnHoldAudioFileId must be provided or set UseDefaultMusicOnHold to true.
+
+The New-CsCallQueue cmdlet may suggest additional steps required to complete the Call Queue setup.
```
Comment (bharatnamburi) [inline skype/skype-ps/skype/New-CsCallQueue.md]: can be removed

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 5]
```diff
@@ -0,0 +1,407 @@
...
+
+## EXAMPLES
+
+### -------------------------- Example 1 -------------------------- 
+```
+New-CsCallQueue -Name "Help Desk" -Domain "litwareinc.com" -UseDefaultMusicOnHold $true
```
Comment (bharatnamburi) [inline skype/skype-ps/skype/New-CsCallQueue.md]: remove Domain

File: skype/skype-ps/skype/Remove-CsCallQueue.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+## SYNTAX
+
+```
+Remove-CsCallQueue -Identity <System.Guid> [-AsJob] [-BypassDualWrite <System.Boolean>] [-Confirm]
+[-DomainController <Object>] [-Force] [-Tenant <System.Guid>] [-WhatIf] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Remove-CsCallQueue cmdlet deletes an existing Call Queue specified by the Identity parameter.
```
Comment (bharatnamburi) [inline skype/skype-ps/skype/Remove-CsCallQueue.md]: May be mention that Remove fails if there are any ApplicationInstances associated with the CallQueue?

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 7]
```diff
@@ -0,0 +1,439 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+applicable: Skype for Business Online
+title: Set-CsCallQueue
+schema: 2.0.0
+author: kenwith
```
Comment (bharatnamburi) [inline skype/skype-ps/skype/Set-CsCallQueue.md]: should we change author?

File: skype/skype-ps/skype/Get-CsCallQueue.md (hunk 3 above)
Comment (mattwlam) [reply skype/skype-ps/skype/Get-CsCallQueue.md]: this was reused from the huntgroup cmdlet markdown files

File: skype/skype-ps/skype/Set-CsCallQueue.md (hunk 7 above)
Comment (mattwlam) [reply skype/skype-ps/skype/Set-CsCallQueue.md]: to what?

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+
+Required: False
+Position: Named
+Default value: 1200
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UseDefaultMusicOnHold
+The UseDefaultMusicOnHold parameter indicates that this Call Queue uses the default music on hold. This parameter cannot be specified together with MusicOnHoldFileName and MusicOnHoldFileContent.
```
Comment (bgams) [inline skype/skype-ps/skype/Set-CsCallQueue.md]: "This parameter cannot be specified together with MusicOnHoldFileName and MusicOnHoldFileContent." Should be MusicOnHoldFileId?

**Pull request merged**